src/plugins/qml/MuseScore/Plugins/PluginsPage.qml

```diff
@@ -184,47 +189,36 @@ Item {
                 }
 
                 onNavigationActivated: function(itemRect) {
-                    Utils.ensureContentVisible(flickable, itemRect, notInstalledPluginsView.headerHeight + 16)
+                    Utils.ensureContentVisible(flickable, itemRect, disabledPluginsView.headerHeight + 16)
                 }
             }
         }
     }
 
-    InstallationPanel {
+    EnablePanel {
         id: panel
 
         property alias selectedPlugin: prv.selectedPlugin
 
         title: Boolean(selectedPlugin) ? selectedPlugin.name : ""
         description: Boolean(selectedPlugin) ? selectedPlugin.description : ""
-        installed: Boolean(selectedPlugin) ? selectedPlugin.installed : false
-        hasUpdate: Boolean(selectedPlugin) ? selectedPlugin.hasUpdate : false
-        neutralButtonTitle: qsTrc("plugins", "View full description")
+        neutralButtonTitle: qsTrc("plugins", "Edit shortcut")
```

Comment:
Better to set this text directly in the EnablePanel implementation. The term "neutral button" is too abstract and we should avoid it